create_skeleton.sh

```diff
@@ -20,12 +20,13 @@ if [[ $release =~ r[1-9].[0-9] ]] && [[ $package_set =~ (dom0|vm) ]] && [[ $dist
 then
     for repo in current current-testing security-testing unstable
     do
+        rm -rf "$release/$repo/$package_set/$dist"
```

Comment:
This is a _very bad_ idea. You try to generate skeleton for freshly added fc42, and surprise - all your packages are gone.

Reply:
Thank you for your feedback. Indeed, this a bad idea if doing on non-fresh clone (that what I assumed in first place).